### PR TITLE
Add SAS URL generation for containers

### PR DESCRIPTION
**Pull request by bear454**

### CODE REVIEW

File: azure_cli/compute_storage_task.py [hunk 1]
```diff
@@ -94,12 +130,38 @@ def process(self):
...
+                self.command_args['--permissions']
+            )
         elif self.command_args['upload']:
             self.__upload()
         elif self.command_args['delete']:
             self.__delete()
         else:
-            raise AzureUnknownStorageCommand(self.command_args)
+            raise AzureUnknownCommand(self.command_args)
+
```
Comment (schaefi) [inline azure_cli/compute_storage_task.py]: The AzureUnknownStorageCommand exception class can then be removed from azurectl_exceptions.py
it's used only here

**Comment by schaefi** [comment]
Could you add some words about this in the man page here:

doc/man/azurectl::compute::storage::container.md

**Comment by schaefi** [comment]
Other than that looks great :)

**Comment by schaefi** [comment]
one thing which came to my mind, did you check if setup.py and package/spec-template contains the correct module requirement list according to the new modules you used for sas support ?

File: azure_cli/compute_storage_task.py (hunk 1 above)
Comment (bear454) [inline azure_cli/compute_storage_task.py]: It wasn't defined in `azurectl_exceptions.py`, or I would have left it be. When I started testing exceptions, it threw an error in `nosetests` as undefined.

Comment (schaefi) [inline azure_cli/compute_storage_task.py]: right, so it was an inconsistency most probably I left behind, this pull request will fix that too which is nice

**Comment by schaefi** [comment]
looks all good now, I'll merge it

**Pull request merged**